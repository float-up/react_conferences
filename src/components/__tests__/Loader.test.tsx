import React from "react";
import {shallow} from "enzyme";
import {Loader} from "../Loader";

let wrapper = shallow(<Loader /> )
describe('Loader', () => {
    it('render Loader', () => {
        expect(wrapper).toMatchSnapshot();
    })
})
