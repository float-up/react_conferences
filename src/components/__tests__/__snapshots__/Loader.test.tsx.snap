// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader render Loader 1`] = `
<div
  className="lds-ellipsis"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
